requirements.txt

```diff
@@ -564,121 +659,111 @@ pydantic==2.11.1 \
-    --hash=sha256:b20ff6e14af8c5e3073304c4ec818a9f20bd92ce9d6d5716982803a5bb0851b6 \
-    --hash=sha256:b30d9d48390deb789c84205fc76ad6c27b00819d60dc955186e9387c55370413 \
-    --hash=sha256:b50cd1cd02c525c44d0b4127833436b491f71a6a2ef93d503e29b0b672fe5acb \
-    --hash=sha256:b5beed6e1f3ee3d20a93b02bdb28dcc94bd192c09f433cd114efcf8ad13829f0 \
-    --hash=sha256:b72c63ed0399fdb7f45b7985025aa851d6c964801b1ad7c58f3ce8f75bc4e070 \
-    --hash=sha256:bb5882ad329cec0b58f006840be4b981938303de5ec72e61160b66f86515bd58 \
-    --hash=sha256:c1444714e5c04dc20ad8246f8fd32b7ef4aef34f2fc7b349f71ae365ceb51e66 \
-    --hash=sha256:c21506d9761d51972d3408c45e38ab44e7bdd244cc7e1e2fcd4af0cd6322cb33 \
-    --hash=sha256:c59c3cadc8a0ae1ac9bed2d7e78f93e71e5d99bba5f9f121c5d7bd41f1faa89a \
-    --hash=sha256:c6fe9cb158200d9a83a38fed3c2cb6766a98adc5f61940ccdd41a75231762e89 \
-    --hash=sha256:c80e6d594122345b41659e5d5ad9cb5d732d81a1da83c336e1b13f55794abafe \
-    --hash=sha256:ca7d1b95b1bca481fb6a8c0ae684a053eef0d6da9e819f6361fd1da7bc1c54ad \
-    --hash=sha256:ca84244021b9d88bd979cb062c58fe7bd00689371b538284f00582781fd1047e \
-    --hash=sha256:d26e1ba192c3207fa2a3d1e049ad7710ace1f09e67ae70e52495db70292cc12d \
-    --hash=sha256:d38504ecbf67b7fe60ae2db028e1b1a54df92a2b899628c82b3edfe885a88059 \
-    --hash=sha256:d74a8e53545bcfd0515754ece2b018fabfb81c0e49d825f8cc1dba09c911f404 \
-    --hash=sha256:d9cb67f2d7d1f129af7e05dbf4fe209f4df043ea1245086bce2a5ee00ca2f5cc \
-    --hash=sha256:dbec17f32828541af5bcebe0ea124235fc620bb5e5d56bf76e38b5b0ad37299d \
-    --hash=sha256:def02d3a0296a171d6723f97cc135ae827bdaca24c9f0665d23ad6f8096334d1 \
-    --hash=sha256:df50a223e1a8e8a066a806267b11ffc47fd797736ffafa70734629305b1b6fe6 \
-    --hash=sha256:e12b71b0ed28cc751e59653d0415aded476f255984dbf0f99d6880342d1911db \
-    --hash=sha256:e19bb6d38d5259b20de62618928fe33d60c4b15975f891503e0703776d23e9f7 \
-    --hash=sha256:e331d7ec5ec582c5078db948eca25d637c0a7f88a037b5016c48cb7c63f7da2c \
-    --hash=sha256:e3bb7a0d84fea2629d10adcf46470ef48ac786976f6c3130a441a260beb4dbad \
-    --hash=sha256:e3da84e67845c83e88c7e96df3e487a16752753f19241d17e887f1f39b7a851c \
-    --hash=sha256:e7126ea9858f9c23a3ec7cd9ecc18be93d1546fe76bc7b331389caf29c4d411f \
-    --hash=sha256:eae6a05d4612d15ba5e22879b710d770d310dbdac6edc53126b86a3be4fe78b5 \
-    --hash=sha256:ec8ce161786818229e96774f4d9159df80589d89b7bc64f3251c57c3fb133289 \
-    --hash=sha256:edc0c6d41431849c6f2f6301f713af730884136c06613a41fd6778446b5bc7c5 \
-    --hash=sha256:ee4491ebcee7f58054585fedeaca653e9822b97445e9b1700ed29846a963f210 \
-    --hash=sha256:f5ee164fb0be1a56c7639794b400815ce36eeb14cdf0a914527c80481b43b43c \
-    --hash=sha256:f7c7ac737d1544615403e8a98956fc02b7505f713c531eab7c4b265618e82af3 \
-    --hash=sha256:ff39a5611d098fcd821dacdfb01dabb434054867d3df646e7b11981a6770a195 \
-    --hash=sha256:fffe3dd67708719a59e82ae36d6b431e2fb4e11b14f36f173c93e448ba963825
+pydantic-core==2.33.0 \
```

Comment:
Not sure why this is downgraded but as far as I can see it is the only one. The rest is removing non-dependencies which were only in requirements.txt